### PR TITLE
Dependency update snakeyaml

### PR DESCRIPTION
**Pull request by RobertAvemarg**
updated snakeyaml dependency and we changed the website from xceptance to posters.xceptance.io and adjusted the slider value in SelenideAddonsTest to make the tests pass again

### CODE REVIEW

**Review by oomelianchuk — APPROVED** [review]
The changes seem good to me. The Jenkins run for this branch was also successful. Thanks @RobertAvemarg for the great work!

File: src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java [hunk 1]
```diff
@@ -36,99 +36,100 @@
...
         });
     }
 
     @Test
     public void testMatchesValueCondition()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
-        $("#search-container .search-field").val("searchphrase").submit();
+        openPostersStartPage();
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java:88]: Use 'mozzarella' as search term and perform similar like in the old example.

File: src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java [hunk 2]
```diff
@@ -36,99 +36,100 @@
...
 
     @Test
     public void testMatchValueCondition()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
-        $("#search-container .search-field").val("searchphrase").submit();
+        openPostersStartPage();
+        $("#header-search-trigger").click();
+        $("#searchForm input").val("bear").submit();
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java:101]: Use 'mozzarella' as search term and perform similar like in the old example. 
(Use ^ and $ in the regex)

File: src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java [hunk 3]
```diff
@@ -36,99 +36,100 @@
...
 
     @Test
     public void testMatchValueConditionError()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
-        $("#search-container .search-field").val("searchphrase").submit();
+        openPostersStartPage();
+        $("#header-search-trigger").click();
+        $("#searchForm input").val("bear").submit();
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java:112]: Use 'mozzarella' as search term and perform similar like in the old example.

File: src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java [hunk 4]
```diff
@@ -36,99 +36,100 @@
...
 
         Assert.assertThrows(ElementShould.class, () -> {
-            $("#content .search-field").should(SelenideAddons.matchValue("\\d+"));
+            $("#searchForm input").should(SelenideAddons.matchValue("\\d+"));
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java:115]: this validation should check the content area e.g. '.header-container #titleSearchText.

**Review by occupant23 — CHANGES_REQUESTED** [review]

**Comment by occupant23** [comment]
@RobertAvemarg I found some small things please adjust.

**Review by occupant23 — APPROVED** [review]

**Comment by occupant23** [comment]
@oomelianchuk Please perform the merge and release it together with @RobertAvemarg

**Pull request merged**